connection.js

```diff
@@ -112,6 +112,7 @@ class Connection {
         this.start_time = Date.now();
         this.last_reject = '';
         this.max_bytes = parseInt(config.get('databytes')) || 0;
+        this.max_mime_parts = parseInt(config.get('max_mime_parts')) || 1000;
```

Comment:
If `parseInt` produces `NaN` this will satisfy the `||` - so move the `|| 1000` into the parseInt.

Also never use parseInt raw like that - always use it with the `, 10` parameter.

And yes I'm aware code you haven't touched doesn't follow either of these rules. Maybe you could fix it all?

Reply:
Perhaps also add the [eslint rule](https://eslint.org/docs/rules/radix)?

Reply:
See [code](https://github.com/haraka/haraka-config/pull/44/files#diff-e22b50e740e102c8de5e858dbcd7753dR52) and [tests](https://github.com/haraka/haraka-config/pull/44/files#diff-447140de3e1b662ff6e9201e7352b529R406) in [haraka-config](
https://github.com/haraka/haraka-config).